contracts/lib/StringHelpers.sol

```diff
@@ -70,8 +70,10 @@ library StringHelpers {
         pure
         returns (bytes)
     {
-        bytes memory numberAsString = new bytes(64);
         uint256 number = uint256(input);
+        bytes memory numberAsString = new bytes(66);
+        numberAsString[0] = byte(48);  // '0'
```

Comment:
Why'd you change the behavior of this to prepend "0x"?

Reply:
Also want to confirm this isn't used by anything except the margin tokens

Reply:
It is only used for the margin tokens